apps/vast/pyvast_threatbus/message_mapping.py

```diff
@@ -4,6 +4,7 @@
 from threatbus.data import ThreatBusSTIX2Constants
 from threatbus.stix2_helpers import is_point_equality_ioc, split_object_path_and_value
 from typing import Tuple, Union
+from re import match
```

Comment:
Please move this to the right place in the imports section (ordered alphabetically).

Reply:
I ran black on the file, kinda confused it does not do that already. Will change.